tools/booster/Proxy.hs

```diff
@@ -165,15 +163,17 @@ respondEither mbStatsVar booster kore req = case req of
                 -- in the old one to work around any potential
                 -- unification bugs.
                 | boosterResult.reason `elem` [Aborted, Stuck] -> do
```

Comment:
what about the branching case?

Reply:
do we want to address that in a separate PR?

Reply:
Yes, this will be addressed in a follow-up PR, along with re-running from pre-branch state after simplification.